_includes/head.html

```diff
@@ -5,7 +5,7 @@
 
     <title>{{ page.title | default: site.title }}</title>
     <meta name="description" content="{{ page.title | default: site.title }}">
-    <link rel="canonical" href="{{ page.url | replace: '/index.html', '' | prepend: site.url }}">
+    <link rel="canonical" href="https://badges-and-achievements.github.io/">
```

Comment:
@nakul-py Hello!

Unfortunately, this is not a solution to the problem.

If you fix the link here, it will be the same on all pages of the site (for example, also on https://badges-and-achievements.github.io/lang/french/ and etc.), and this is not true.